agent/acl_endpoint.go

```diff
@@ -40,40 +39,18 @@ func (s *HTTPHandlers) ACLBootstrap(resp http.ResponseWriter, req *http.Request)
 	args := structs.DCSpecificRequest{
 		Datacenter: s.agent.config.Datacenter,
 	}
-
-	legacy := false
-	legacyStr := req.URL.Query().Get("legacy")
```

Comment:
Should also update the website to indicate that it stopped working in 1.11.

Reply:
Ya, good call. I looked at https://www.consul.io/api-docs/acl#bootstrap-acls, and didn't see this query parameter documented. Maybe it's hidden somewhere else?  I plan to do another full pass of all of these at the end.